router/v3/qall.go

```diff
@@ -0,0 +1,470 @@
+package v3
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/gofrs/uuid"
+	"github.com/labstack/echo/v4"
+	"github.com/livekit/protocol/auth"
+	"github.com/livekit/protocol/livekit"
+	"github.com/livekit/protocol/webhook"
+	lksdk "github.com/livekit/server-sdk-go/v2"
+	"github.com/traPtitech/traQ/model"
+	"github.com/traPtitech/traQ/router/extension/herror"
+	"github.com/traPtitech/traQ/service/qall"
+	"go.uber.org/zap"
+)
+
+// GetQallEndpoints
+func (h *Handlers) GetQallEndpoints(c echo.Context) error {
+	return c.JSON(http.StatusOK, map[string]string{
+		"endpoint": h.Config.LiveKitHost,
+	})
+}
+
+// GetSoundboardItems
+func (h *Handlers) GetSoundboardItems(c echo.Context) error {
+	items, err := h.Repo.GetAllSoundboardItems()
+	if err != nil {
+		return herror.InternalServerError(err)
+	}
+	return c.JSON(http.StatusOK, items)
+}
+
+// CreateSoundboardItem
+func (h *Handlers) CreateSoundboardItem(c echo.Context) error {
+	src, uploadedFile, err := c.Request().FormFile("file")
+	if err != nil {
+		return herror.BadRequest(err)
+	}
+	defer src.Close()
+	if uploadedFile.Size == 0 {
+		return herror.BadRequest("non-empty file is required")
+	}
+
+	mimeType := uploadedFile.Header.Get(echo.HeaderContentType)
+	soundName := c.FormValue("name")
+	creatorID := uuid.FromStringOrNil(c.FormValue("creatorId"))
+	stampID := uuid.FromStringOrNil(c.FormValue("stampId"))
+
+	if err := h.Soundboard.SaveSoundboardItem(uuid.Must(uuid.NewV7()), soundName, mimeType, model.FileTypeSoundboardItem, src, &stampID, creatorID); err != nil {
+		return herror.InternalServerError(err)
+	}
+
+	return c.NoContent(http.StatusNoContent)
+}
+
+// PlaySoundboardItem
+func (h *Handlers) PlaySoundboardItem(c echo.Context) error {
+	type playSoundboardItemRequest struct {
+		SoundID string `json:"soundId"`
+		RoomID  string `json:"roomId"`
+	}
+	var req playSoundboardItemRequest
```

Comment:
一度しか出てこない構造体の場合、他のファイルでは命名せずに構造体を使っていることが多いようでした
それに合わせると、 ↓ のような書き方になると思います
```suggestion
	var req struct {
		SoundID string `json:"soundId"`
		RoomID  string `json:"roomId"`
	}
```